docs/howtos/airgap/03-hauler.md

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_label: Hauler installation
+title: Air gap installation with Hauler
+description: Air gap installation for Kubewarden using Hauler
+keywords: [kubewarden, kubernetes, air gap installation, hauler]
+doc-persona: [kubewarden-operator, kubewarden-integrator]
+doc-type: [howto]
+doc-topic: [operator-manual, air gap, installation, hauler]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/howtos/airgap/hauler-sync"/>
+</head>
+
+This guide shows you how to install Kubewarden in air-gapped environments using
+[Hauler](https://docs.hauler.dev/docs/intro). Hauler is a tool that helps
+users run workloads in air-gapped environments. It moves the required 
+resources for apps into those air-gapped environments.
+
+Kubewarden uses Hauler capabilities by providing a manifest
+file with all the required resources to run Kubewarden in a private environment. This
+documentation describes how you can use it. Refer to the
+Hauler documentation to learn more about it.
+
+The basic workflow using the Hauler manifest is:
+
+- to get it from a Kubewarden release page
+- load all the resources required to run Kubewarden into a local store
+- export them to a file
```

Comment:
I suppose this means exporting the local store into a file, and not the resources?